src/widgets/RainWidget.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import GenericWidget from './GenericWidget';
+import './RainWidget.css';
+
+class RainWidget extends Component {
+
+  render() {
+    return (
+      <GenericWidget class="RainWidget" icon="cloud"
+          isOn={this.props.rain > 0}
+          value1={<div className="rain">{this.props.rain}</div>}
+          value2={<div className="rate">{this.props.rate}</div>}
```

Comment:
BTW I realize that we're mixin css2 (:after) and css3 (::after) notation. Both are accepted but it's a little messy. I'll do a pass on the codebase to use ::after everywhere.